js/list-photo.js

```diff
@@ -0,0 +1,42 @@
+import {debounce} from './util.js';
+import {renderThumbnails} from './thumbnails.js';
+let picture = [];
+const PPORTION_PFOTOS = 10;
+const filtersContainer = document.querySelector('.img-filters');
+const actieButton = 'img-filters__button--active';
+
+const discussedFilters = filtersContainer.querySelector('filter-discussed');
+const randomFilters = filtersContainer.querySelector('filter-random');
+const defaultFilters = filtersContainer.querySelector('filter-default');
+
+const debounceRender = debounce(renderThumbnails);
+
+function filterChange (evt) {
+  const activeFilter = document.querySelector(`.${actieButton}`);
+  if (activeFilter === evt.target) {
+    return;
+  }
+  activeFilter.classList.remove(actieButton);
+  evt.target.classList.add(actieButton);
+  selectFilter();
+}
+
+let filteredPhoto = [];
+function selectFilter () {
+  filtersContainer.addEventListener('click', (evt) => {
+    if(evt.target.id === defaultFilters){
```

Comment:
Ты же искала элементы, значит нужно сравнивать их, а не id
```suggestion
    if(evt.target === defaultFilters){
```